src/community/geofence-server/src/main/java/org/geoserver/geofence/rest/UsersRestController.java

```diff
@@ -0,0 +1,151 @@
+package org.geoserver.geofence.rest;
```

Comment:
Should there perhaps be a gs-rest-sec, and then any rest for the default security framework can be put there? And keep the geofence rest in this one